pages/ledger/index.vue

```diff
@@ -9,6 +9,33 @@
         @click.native="connect"
       />
     </div>
+    <div v-if="isWindows && !hasHIDEnabled">
+      Due to recent Ledger updates, using a Ledger on Windows now requires
+      "Experimental Web Platform features" to be enabled.
+      <div v-if="isChrome">
+        <br />
+        <br />
+        <p>
+          Please copy the link below into a new tab and set the "Experimental
+          Web Platform features" flag to "Enabled":
+        </p>
+        <div
```

Comment:
please refactor into a copyable link... this is just duplicated code

Reply:
you mean CopyButton? I don't see any other reusable component for this